rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_23.md

```diff
@@ -1,41 +1,43 @@
-# Proposal to upgrade the ledger suite orchestrator canister to add ckXAUT
+# Proposal to upgrade the ledger suite orchestrator canister
```

Comment:
This is actually a new file, ignore the diff from the left. Git is confused with the rename from this [commit](https://github.com/dfinity/ic/pull/1075/commits/975a00a588a8e2aad3adfcd578c1fe916ca11106).